etc/api/angular_devkit/architect/src/index.d.ts

```diff
@@ -0,0 +1,102 @@
+export declare class Architect {
```

Comment:
If you instance you changed the schematics public API, you'd need to regenerate the golden file. This can be done via a bazel command example;

```
$ bazel run //etc/api:angular_devkit_schematics_api.accept
```

Note, I don't think this will run on Windows as under the hook it will create and use bash files.

Reply:
Thank you for the explanation.

Tried the commands on Windows and below are the results:
```
$ bazel-16.1 run //etc/api:angular_devkit_schematics_api.accept
INFO: Analysed target //etc/api:angular_devkit_schematics_api.accept (0 packages loaded).
INFO: Found 1 target...
Target //etc/api:angular_devkit_schematics_api.accept up-to-date:
  C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_api.accept_loader.js
  C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_api.accept.sh
INFO: Elapsed time: 2.212s, Critical Path: 0.10s
INFO: 0 processes.
INFO: Build completed successfully, 1 total action
INFO: Running command line: C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuildINFO: Build completed successfully, 1 total action
C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_api.accept.sh: line 37: grep: command not found
C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_api.accept.sh: line 37: cut: command not found
C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_api.accept.sh: line 37: : No such file or directory
```
```
$ bazel-16.1 run //etc/api:angular_devkit_schematics_tasks_api.accept
INFO: Analysed target //etc/api:angular_devkit_schematics_tasks_api.accept (0 packages loaded).
INFO: Found 1 target...
Target //etc/api:angular_devkit_schematics_tasks_api.accept up-to-date:
  C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_tasks_api.accept_loader.js
  C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_tasks_api.accept.sh
INFO: Elapsed time: 2.583s, Critical Path: 0.09s
INFO: 0 processes.
INFO: Build completed successfully, 1 total action
INFO: Running command line: C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuildINFO: Build completed successfully, 1 total action
C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_tasks_api.accept.sh: line 37: grep: command not found
C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_tasks_api.accept.sh: line 37: cut: command not found
C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_tasks_api.accept.sh: line 37: : No such file or directory
```
```
$ bazel-16.1 test //etc/api:angular_devkit_schematics_tasks_api
INFO: Analysed target //etc/api:angular_devkit_schematics_tasks_api (2 packages loaded).
INFO: Found 1 test target...
FAIL: //etc/api:angular_devkit_schematics_tasks_api (see C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/testlogs/etc/api/angular_devkit_schematics_tasks_api/test.log)
Target //etc/api:angular_devkit_schematics_tasks_api up-to-date:
  C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_tasks_api_loader.js
  C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/bin/etc/api/angular_devkit_schematics_tasks_api.sh
INFO: Elapsed time: 3.588s, Critical Path: 1.29s
INFO: 1 process: 1 local.
INFO: Build completed, 1 test FAILED, 2 total actions
//etc/api:angular_devkit_schematics_tasks_api                            FAILED in 1.1s
  C:/users/kamik/_bazel_kamik/gq7cl633/execroot/angular_devkit/bazel-out/x64_windows-fastbuild/testlogs/etc/api/angular_devkit_schematics_tasks_api/test.log

INFO: Build completed, 1 test FAILED, 2 total actions
```

So no, TS API guardian does not run on Windows.